data/transportation/roads-system/index.html

```diff
@@ -13,36 +13,28 @@
 categories: []
 date: 2011-08-25 23:12:40 -0600
 HighwayLinearReferencingSystemRoutes:
-  downloads:
-    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mSURhZHpJMDFnM1U&export=download
-      label: 'UDOT LRS Routes: File Geodatabase'
-    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mV2VRNTduVXNyMjg&export=download
-      label: 'UDOT LRS Routes: Shapefile'
+  hub:
+    name: Utah UDOT Routes LRS
+    item_id: cbc11cc11baf4160bc323fcb9384cc8a
```

Comment:
**question**: wasn't this replaced by udot roads & highways?

Reply:
i think we talked about it, but i'm not sure it happened.  I'll reach out to udot today and get that conversation going again. but, yes - it should be.  it will have to go through the 'farm to agol' process.

Reply:
@stdavis are you still using this layer for WVC or other?